### PR TITLE
Port triple-slash comments to docs for Microsoft.Extensions.Http

### PR DESCRIPTION
**Pull request by CarnaViire**
Port docs added in https://github.com/dotnet/runtime/pull/57719

Contributes to https://github.com/dotnet/runtime/issues/43916

### CODE REVIEW

**Comment by gewarren** [comment]
Docs Build status updates of commit _[bc8109c](https://github.com/CarnaViire/dotnet-api-docs/commits/bc8109c2920f2ca53eb51362b78432c1d4dc1ec1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/Microsoft.Extensions.Http.Logging/LoggingHttpMessageHandler.xml](https://github.com/CarnaViire/dotnet-api-docs/blob/thhpclientfactory-docs-port/xml/Microsoft.Extensions.Http.Logging/LoggingHttpMessageHandler.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.http.logging.logginghttpmessagehandler?view=dotnet-plat-ext-2.1&branch=pr-en-us-7103) |
[xml/Microsoft.Extensions.Http.Logging/LoggingScopeHttpMessageHandler.xml](https://github.com/CarnaViire/dotnet-api-docs/blob/thhpclientfactory-docs-port/xml/Microsoft.Extensions.Http.Logging/LoggingScopeHttpMessageHandler.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.http.logging.loggingscopehttpmessagehandler?view=dotnet-plat-ext-2.1&branch=pr-en-us-7103) |
[xml/Microsoft.Extensions.Http/HttpMessageHandlerBuilder.xml](https://github.com/CarnaViire/dotnet-api-docs/blob/thhpclientfactory-docs-port/xml/Microsoft.Extensions.Http/HttpMessageHandlerBuilder.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.http.httpmessagehandlerbuilder?view=dotnet-plat-ext-2.1&branch=pr-en-us-7103) |
[xml/Microsoft.Extensions.Http/IHttpMessageHandlerBuilderFilter.xml](https://github.com/CarnaViire/dotnet-api-docs/blob/thhpclientfactory-docs-port/xml/Microsoft.Extensions.Http/IHttpMessageHandlerBuilderFilter.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.http.ihttpmessagehandlerbuilderfilter?view=dotnet-plat-ext-2.1&branch=pr-en-us-7103) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202109031354367188-7103%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=3TK1xmIj4K7k0GklSt9%2BvygEY1H5hLEnzPgS%2B5tiV7k%3D&st=2021-09-03T14%3A23%3A06Z&se=2021-10-04T14%3A28%3A06Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/500c749d-de78-3d4a-00e4-269d599bf9ec?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: xml/Microsoft.Extensions.Http.Logging/LoggingHttpMessageHandler.xml [hunk 1]
```diff
@@ -108,7 +112,15 @@
         <param name="cancellationToken">To be added.</param>
         <summary>To be added.</summary>
         <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Loggs the request to and response from the sent <xref:System.Net.Http.HttpRequestMessage>.
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Http.Logging/LoggingHttpMessageHandler.xml:120]:
```suggestion
Logs the request and response from the sent <xref:System.Net.Http.HttpRequestMessage>.
```

File: xml/Microsoft.Extensions.Http.Logging/LoggingHttpMessageHandler.xml [hunk 2]
```diff
@@ -108,7 +112,15 @@
...
+
+          ]]></format>
+        </remarks>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Http.Logging/LoggingHttpMessageHandler.xml:123]:
```suggestion
        </remarks>
        <inheritdoc />
```

File: xml/Microsoft.Extensions.Http.Logging/LoggingScopeHttpMessageHandler.xml [hunk 3]
```diff
@@ -108,7 +112,15 @@
         <param name="cancellationToken">To be added.</param>
         <summary>To be added.</summary>
         <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Loggs the request to and response from the sent <xref:System.Net.Http.HttpRequestMessage>.
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Http.Logging/LoggingScopeHttpMessageHandler.xml:120]:
```suggestion
Logs the request and response from the sent <xref:System.Net.Http.HttpRequestMessage>.
```

File: xml/Microsoft.Extensions.Http.Logging/LoggingScopeHttpMessageHandler.xml [hunk 4]
```diff
@@ -108,7 +112,15 @@
...
+
+          ]]></format>
+        </remarks>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Http.Logging/LoggingScopeHttpMessageHandler.xml:123]:
```suggestion
        </remarks>
        <inheritdoc />
```

File: xml/Microsoft.Extensions.Http/HttpMessageHandlerBuilder.xml [hunk 5]
```diff
@@ -129,11 +129,17 @@
...
+        <returns>The HTTP message handler chain.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="primaryHandler " /> or <paramref name="additionalHandlers " /> is <see langword="null" />.</exception>
+        <exception cref="T:System.InvalidOperationException">
+          <paramref name="additionalHandlers " /> contains a <see langword="null" /> entry.
+            -or- The <c>DelegatingHandler.InnerHandler</c> property must be <see langword="null" />. <c>DelegatingHandler</c> instances provided to <c>HttpMessageHandlerBuilder</c> must not be reused or cached.</exception>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Http/HttpMessageHandlerBuilder.xml:142]:
```suggestion

-or-

<c>DelegatingHandler.InnerHandler</c> is not <see langword="null" />. <c>DelegatingHandler</c> instances provided to <c>HttpMessageHandlerBuilder</c> must not be reused or cached.</exception>
```

**Review by gewarren — COMMENTED** [review]
I left some suggestions.

**Comment by gewarren** [comment]
Docs Build status updates of commit _[8b0110f](https://github.com/CarnaViire/dotnet-api-docs/commits/8b0110f6e01289660db56264e6f79379c9fa80fc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/Microsoft.Extensions.Http.Logging/LoggingHttpMessageHandler.xml](https://github.com/CarnaViire/dotnet-api-docs/blob/thhpclientfactory-docs-port/xml/Microsoft.Extensions.Http.Logging/LoggingHttpMessageHandler.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.http.logging.logginghttpmessagehandler?view=dotnet-plat-ext-2.1&branch=pr-en-us-7103) |
[xml/Microsoft.Extensions.Http.Logging/LoggingScopeHttpMessageHandler.xml](https://github.com/CarnaViire/dotnet-api-docs/blob/thhpclientfactory-docs-port/xml/Microsoft.Extensions.Http.Logging/LoggingScopeHttpMessageHandler.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.http.logging.loggingscopehttpmessagehandler?view=dotnet-plat-ext-2.1&branch=pr-en-us-7103) |
[xml/Microsoft.Extensions.Http/HttpMessageHandlerBuilder.xml](https://github.com/CarnaViire/dotnet-api-docs/blob/thhpclientfactory-docs-port/xml/Microsoft.Extensions.Http/HttpMessageHandlerBuilder.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.http.httpmessagehandlerbuilder?view=dotnet-plat-ext-2.1&branch=pr-en-us-7103) |
[xml/Microsoft.Extensions.Http/IHttpMessageHandlerBuilderFilter.xml](https://github.com/CarnaViire/dotnet-api-docs/blob/thhpclientfactory-docs-port/xml/Microsoft.Extensions.Http/IHttpMessageHandlerBuilderFilter.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.http.ihttpmessagehandlerbuilderfilter?view=dotnet-plat-ext-2.1&branch=pr-en-us-7103) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C6%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202109061715145449-7103%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=vpmzbzV2cdrGoPjZ016fWtRtWJkawVZ528Wkz8d0fNk%3D&st=2021-09-06T17%3A43%3A26Z&se=2021-10-07T17%3A48%3A26Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/500c749d-de78-3d4a-00e4-269d599bf9ec?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by gewarren — APPROVED** [review]
LGTM. I checked that inheritdoc is working for SendAsync.

**Review by carlossanlop — APPROVED** [review]

**Pull request merged**